x-pack/plugins/cases/server/services/notifications/email_notification_service.test.ts

```diff
@@ -0,0 +1,197 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { createCasesClientMockArgs } from '../../client/mocks';
+import { userProfiles } from '../../client/user_profiles.mock';
+import { mockCases } from '../../mocks';
+import { EmailNotificationService } from './email_notification_service';
+
+describe('EmailNotificationService', () => {
+  const clientArgs = createCasesClientMockArgs();
+  const sendPlainTextEmail = jest.fn();
+  const isEmailServiceAvailable = jest.fn();
+  const caseSO = mockCases[0];
+  const assignees = userProfiles.map((userProfile) => ({ uid: userProfile.uid }));
+
+  const notifications = {
```

Comment:
Just curious, does the notification plugin return a mock that we could leverage here?

Reply:
Good catch! When I wrote the code they did not but they added it later. I will update to include it.